### PR TITLE
[PIL-682][OK] - Fix Cross site scripting issue on remaining screens

### PR DESCRIPTION
**Pull request by olakunlekuye2**

### CODE REVIEW

**Review by AKKUM — DISMISSED** [review]

File: conf/messages.en [hunk 1]
```diff
@@ -222,6 +222,8 @@ upeNameRegistration.title = What is the name of the ultimate parent entity?
 upeNameRegistration.heading = What is the name of the ultimate parent entity?
 upeNameRegistration.error.required = You need to enter the name of the ultimate parent entity
 upeNameRegistration.error.length = The name of the ultimate parent entity must be 105 characters or less
+upeNameRegistration.error.invalid = The name of the person must only include letters a to z, numbers 0 to 9, ampersands (&), apostrophes, commas, forward slashes, full stops, hyphens, round brackets and spaces
```
Comment (jassalrichy) [inline conf/messages.en:225]: The name of the **ultimate parent entity** must ...

File: conf/messages.en [hunk 2]
```diff
@@ -383,6 +385,8 @@ nfmNameRegistration.checkYourAnswersLabel = Name
 nfmNameRegistration.error.required =  You need to enter the name of the nominated filing member
 nfmNameRegistration.error.length = The name of the nominated filing member must be 105 characters or less
 nfmNameRegistration.change.hidden = What is the name of the nominated filing member?
+nfmNameRegistration.error.invalid = The name of the person must only include letters a to z, numbers 0 to 9, ampersands (&), apostrophes, commas, forward slashes, full stops, hyphens, round brackets and spaces
```
Comment (jassalrichy) [inline conf/messages.en:388]: The name of the **nominated filing member** must ...

File: conf/messages.en (hunk 1 above)
Comment (olakunlekuye2) [reply conf/messages.en:225]: According to the ticket : ultimate-parent/no-id/input-name - The name of the person must only include letters a to z, numbers 0 to 9, ampersands (&), apostrophes, commas, forward slashes, full stops, hyphens, round brackets and spaces

File: conf/messages.en (hunk 2 above)
Comment (olakunlekuye2) [reply conf/messages.en:388]: According to the ticket : filing-member/no-id/input-name - The name of the person must only include letters a to z, numbers 0 to 9, ampersands (&), apostrophes, commas, forward slashes, full stops, hyphens, round brackets and spaces

File: conf/messages.en [hunk 3]
```diff
@@ -644,6 +650,7 @@ secondaryContactName.checkYourAnswersLabel = Second contact name
 secondaryContactName.hint = For example, ‘Tax team’ or ‘Ashley Smith’.
 secondaryContactName.error.required = Enter name of the person of team we should contact
 secondaryContactName.error.length = The name of the contact person or team should be 160 characters or less
```
[the remark this answers is not in the document]
Comment (olakunlekuye2) [reply conf/messages.en:652]: This was not in the ticket to change

File: test/forms/NfmNameRegistrationFormProviderSpec.scala [hunk 4]
```diff
@@ -21,11 +21,11 @@ import play.api.data.FormError
...
 class NfmNameRegistrationFormProviderSpec extends StringFieldBehaviours {
 
-  val requiredKey = "nfmNameRegistration.error.required"
-  val lengthKey   = "nfmNameRegistration.error.length"
-  val maxLength   = 105
-
-  val form = new NfmNameRegistrationFormProvider()()
+  val requiredKey  = "nfmNameRegistration.error.required"
+  val lengthKey    = "nfmNameRegistration.error.length"
+  val maxLength    = 105
```
[the remark this answers is not in the document]
Comment (olakunlekuye2) [reply test/forms/NfmNameRegistrationFormProviderSpec.scala:26]: Done

**Review by jassalrichy — APPROVED** [review]

**Pull request merged**